src/pages/rechner/[frage].tsx

```diff
@@ -0,0 +1,122 @@
+import { GetStaticPaths, GetStaticProps, NextPage } from 'next';
+import { Page } from '../../layouts/page';
+import React, { useEffect, useState } from 'react';
+import { InputField } from '../../elements/input-field';
+import { Copy } from '../../identity/copy';
+import { Heading2 } from '../../identity/heading-2';
+import { Question } from '../../data/questions';
+import Questions from '../../data/questions.json';
+import { Heading1 } from '../../identity/heading-1';
+import { getLocalStorage, setLocalStorage } from '../../utils/local-storage';
+import { LinkElement } from '../../elements/link';
+
+type Props = {
+  question: Question;
+  MAX_QUESTION_NUMBER: number;
+};
+
+//later into data file or DB
+const theory = {
+  title: 'Facts/Transparenz über diese Kategorie und oder Frage',
+  content:
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Error neque odio explicabo, necessitatibus eaque numquam tenetur deleniti sequi at facere earum eos, voluptates culpa nam, quae exercitationem recusandae? Aspernatur, aliquam.',
+};
+
+//nurno d Frag inegeh,
+const Frage: NextPage<Props> = ({ question: { id, title, label, emissionfactor, initialAnswer }, MAX_QUESTION_NUMBER }) => {
+  const [answer, setAnswer] = useState('');
+  const [impact, setImpact] = useState('');
+
+  useEffect(() => {
+    if (getLocalStorage(id) !== undefined) {
+      setAnswer(getLocalStorage(id) || '');
+      setImpact(getLocalStorage('impact' + id) || '');
+      saveCurrentQuestionIntoLocalStorage();
+    }
+  }, [id]);
+
+  const saveCurrentQuestionIntoLocalStorage = () => {
+    setLocalStorage(id, answer);
+    setLocalStorage('impact' + id, impact);
```

Comment:
Would probably be safer to create a constant (like `const LOCALSTORAGE_IMPACT_KEY = 'impact'`) for your localstorage keys since you are already using `'impact'` twice and if you change it in one place you might forget to change it in the second.